add new flags for redline testing (#2)
OVI3D0/project-website: _posts/2025-04-24-Redline-Testing-Comes-to-OpenSearch-Benchmark.md

  --redline-test
```

> Users can customize redline test parameters such as the maximum number of clients, the client ramp-up rate, the percentage of clients to pause during back-off, and the wait time before resuming scale-up with the following flags:

Reviewer: ```suggestion
Users can customize redline test parameters—such as the maximum number of clients, the client ramp-up rate, the percentage of clients to pause during back-off, and the wait time before resuming scale-up—using the following flags:

```

> `--redline-scale-step`: controls how many clients to unpause per scaling iteration

Reviewer: ```suggestion
`--redline-scale-step`: Specifies the number of clients to unpause in each scaling iteration.
```

> `--redline-scaledown-percentage`: controls what percentage of clients to pause when an error occurs

Reviewer: ```suggestion
`--redline-scaledown-percentage`: Specifies the percentage of clients to pause when an error occurs.
```

> `--redline-post-scaledown-sleep`: controls how many seconds the feedback actor waits before scaling up again after a scale down

Reviewer: ```suggestion
- `--redline-post-scaledown-sleep`: Specifies the number of seconds the feedback actor waits before initiating a scale-up after scaling down.
```

> `--redline-max-clients`: controls the max number of clients to allow during redline testing. If this one is unset, OSB will default to the number of clients defined in the test procedure

Reviewer: ```suggestion
- `--redline-max-clients`: Specifies the maximum number of clients allowed during redline testing. If unset, OpenSearch Benchmark defaults to the number of clients defined in the test procedure.
```
